src/base/bittorrent/trackerentry.cpp

```diff
@@ -49,15 +49,16 @@ TrackerEntry::TrackerEntry(const lt::announce_entry &nativeEntry)
 
 QString TrackerEntry::url() const
 {
-    return QString::fromStdString(m_nativeEntry.url);
+    return QString::fromStdString(nativeEntry().url);
 }
 
 bool TrackerEntry::isWorking() const
 {
 #if (LIBTORRENT_VERSION_NUM < 10200)
-    return m_nativeEntry.is_working();
+    return nativeEntry().is_working();
```

Comment:
Won't all these `nativeEntry()` produce unnecessary temp copies?

Reply:
I'll change `nativeEntry()` to return const reference of `m_nativeEntry`.